docs/examples/count-tokens.md

```diff
@@ -1,4 +1,4 @@
-To count tokens you can use Griptape events and the `TokenCounter` util:
+To count tokens you can use Griptape events and the [TokenCounter](../reference/griptape/utils/token_counter.md) util:
```

Comment:
Wondering if we should setup some kind of dead/broken link integration in case we change doc structure but forget to update links in other places.

Reply:
@vasinov The Read the Docs build now [fails on warning](https://github.com/griptape-ai/griptape-docs/pull/77/files#diff-cde814ef2f549dc093f5b8fc533b7e8f47e7b32a8081e0760e57d5c25a1139d9R18). Example [here](https://readthedocs.org/projects/griptape/builds/21888619/).